internal/verification/database/health_authority_db.go

```diff
@@ -54,11 +54,11 @@ func (db *HealthAuthorityDB) AddHealthAuthority(ctx context.Context, ha *model.H
 		row := tx.QueryRow(ctx, `
 			INSERT INTO
 				HealthAuthority
-				(iss, aud, name)
+				(iss, aud, name, jwks_uri)
```

Comment:
This is a 1-1 mapping of a JWK URI and a PHA. Do we envision a world where one PHA could have multiple JWK URIs, maybe during a migraiton? @mikehelmick

Reply:
I'm going to say no to needing multiple.

If someone is migrating from one verification server to another, they will have to use the existing key management.

Seems like the right complexity tradeoff here.